utils/liquibase.js

```diff
@@ -169,7 +165,8 @@ function parseLiquibaseLoadColumnType(entity, field) {
     return columnType;
   }
 
-  if (columnType === 'blob' || columnType === 'longblob') {
+  // eslint-disable-next-line no-template-curly-in-string
```

Comment:
If columnType is stored, this will break.

Reply:
Strangely, It takes the liquibase columnType instead of the jdl one, so this one pass!